README.md

```
@@ -7,54 +7,56 @@
 ## ⚙️ 기술스택
 
 - ### **Frontend**
 
-<img src="https://img.shields.io/badge/React-61dafb?style=for-the-badge&logo=React&logoColor=white">
-<img src="https://img.shields.io/badge/TypeScript-3178c6?style=for-the-badge&logo=TypeScript&logoColor=white">
-<img src="https://img.shields.io/badge/ReactQuery-ff4154?style=for-the-badge&logo=ReactQuery&logoColor=white">
-<img src="https://img.shields.io/badge/ReactRouter-ca4245?style=for-the-badge&logo=ReactRouter&logoColor=white">
-<img src="https://img.shields.io/badge/VanillaExtract-f786ad?style=for-the-badge&logo=VanillaExtract&logoColor=white">
-<img src="https://img.shields.io/badge/Vite-646cff?style=for-the-badge&logo=Vite&logoColor=white">
-<img src="https://img.shields.io/badge/Normalize.css-E3695F?style=for-the-badge&logo=Normalize.css&logoColor=white">
+<img src="https://img.shields.io/badge/React-61dafb?style=for-the-badge&logo=React&logoColor=white"> <img src="https://img.shields.io/badge/TypeScript-3178c6?style=for-the-badge&logo=TypeScript&logoColor=white"> <img src="https://img.shields.io/badge/ReactQuery-ff4154?style=for-the-badge&logo=ReactQuery&logoColor=white"> <img src="https://img.shields.io/badge/ReactRouter-ca4245?style=for-the-badge&logo=ReactRouter&logoColor=white"> <img src="https://img.shields.io/badge/VanillaExtract-f786ad?style=for-the-badge&logo=VanillaExtract&logoColor=white"> <img src="https://img.shields.io/badge/Vite-646cff?style=for-the-badge&logo=Vite&logoColor=white"> <img src="https://img.shields.io/badge/Normalize.css-E3695F?style=for-the-badge&logo=Normalize.css&logoColor=white">
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)